tests/kernel/context/src/main.c

```diff
@@ -697,6 +697,16 @@ static void busy_wait_thread(void *mseconds, void *arg2, void *arg3)
 	k_busy_wait(usecs);
 	TC_PRINT("Thread busy waiting completed\n");
 
+	/* FIXME: Broken on Nios II, see #22956 */
+#ifndef CONFIG_NIOS2
```

Comment:
We should exclude `CONFIG_LITEX_TIMER` here too since it is known to be broken (see https://github.com/zephyrproject-rtos/zephyr/issues/23519#issuecomment-601252378 and #23622).